crates/chainio/clients/avsregistry/src/writer.rs

```diff
@@ -284,13 +317,141 @@ impl AvsRegistryChainWriter {
 
         let contract_call = contract_registry_coordinator.deregisterOperator(quorum_numbers);
 
-        let tx_result = contract_call.send().await;
-        match tx_result {
-            Ok(tx) => {
-                info!(tx_hash = ?tx,"succesfully deregistered operator with the AVS's registry coordinator" );
-                Ok(*tx.tx_hash())
-            }
-            Err(e) => Err(AvsRegistryError::AlloyContractError(e)),
-        }
+        let tx = contract_call
+            .send()
+            .await
+            .map_err(AvsRegistryError::AlloyContractError)?;
+        info!(tx_hash = ?tx,"succesfully deregistered operator with the AVS's registry coordinator" );
+        Ok(*tx.tx_hash())
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::AvsRegistryChainWriter;
+    use alloy_node_bindings::Anvil;
+    use alloy_primitives::{Address, Bytes, FixedBytes, U256};
+    use alloy_provider::Provider;
+    use eigen_crypto_bls::BlsKeyPair;
+    use eigen_logging::get_test_logger;
+    use eigen_testing_utils::anvil_constants::{
+        get_operator_state_retriever_address, get_registry_coordinator_address, ANVIL_RPC_URL,
+    };
+    use std::time::Duration;
+    use tokio::time::sleep;
+
+    const ANVIL_HTTP_URL: &str = "http://localhost:8545";
```

Comment:
https://github.com/Layr-Labs/eigensdk-rs/blob/2d455ccb25743f5f79aef56a20e85adf0e330f95/testing/testing-utils/src/anvil_constants.rs#L20

Reply:
done! 
also generalized all this constants of the whole project in the file anvil_constants.rs